### PR TITLE
[1LP][RFR] Remove pytest_selenium

### PR DESCRIPTION
**Pull request by psav**
Final removal of the old

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit af13c52ec8b3feb41e43a587f8b5a9ab4424f587:

`fixtures/browser.py`:
- :heavy_exclamation_mark: Line 136:34: **W292** *no newline at end of file*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/common/__init__.py [hunk 1]
```diff
@@ -1,20 +1,25 @@
...
-from cfme.utils.units import Unit
+from cfme.utils.appliance import current_appliance
+from cfme.utils.appliance.implementations.ui import navigate_to, navigator, CFMENavigateStep
 from cfme.utils.log import logger
+from cfme.utils.units import Unit
+from widgetastic_manageiq import BaseNonInteractiveEntitiesView, CheckableManageIQTree, BreadCrumb
+
+
+def sel():
+    return current_appliance.broeser.widgetastic.selenium
```
Comment (izapolsk) [inline cfme/common/__init__.py]: typo

File: cfme/tests/configure/test_default_filters.py [hunk 2]
```diff
@@ -38,45 +46,60 @@ def test_infrastructurehost_defaultfilters(appliance):
...
+def test_infrastructuretemplates_defaultfilters(appliance):
     filters = [['Infrastructure', 'Virtual Machines', 'Templates', 'Platform / Redhat']]
     df = st.DefaultFilter(name='Platform / Redhat')
     df.update({'filters': [(k, True) for k in filters]})
     navigate_to(vms.Template, 'TemplatesOnly')
-    assert sel.is_displayed_text(df.name), "Default Filter settings Failed!"
+    loc = "//*[normalize-space(text())={}]".format(quoteattr(df.name))
+    assert appliance.browser.widgetastic.sel.is_displayed_text(
+        loc
+    ), "Default Filter settings Failed!"
```
Comment (ohiliazo) [inline cfme/tests/configure/test_default_filters.py]: @psav, why don't use this instead?
```python
view = navigate_to(vms.Template, 'TemplatesOnly')
assert view.sidebar.images.tree.has_path('All Images', 'Global Filters', 'Platform / Redhat'), "Default Filter settings Failed!"
```

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

File: cfme/tests/cloud/test_stack.py [hunk 3]
```diff
@@ -83,7 +82,8 @@ def test_outputs_link_url(stack):
         # TODO: Need to come back to this one
         table = Table('//div[@id="list_grid"]//table[contains(@class, "table-selectable")]')
```
Comment (mfalesni) [inline cfme/tests/cloud/test_stack.py]: This one will not work, it is not instantiated properly.

File: cfme/tests/configure/test_session_timeout.py [hunk 4]
```diff
@@ -27,10 +26,12 @@ def _finalize():
     # Try getting timeout
     # I had to use wait_for because on 5.4 and upstream builds it made weird errors
     wait_for(
-        lambda: sel.elements(
+        lambda: appliance.browser.widgetastic.selenium.elements(
```
Comment (mfalesni) [inline cfme/tests/configure/test_session_timeout.py]: We should probably introduce a fixture for getting this (later).

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Pull request merged**